tests/common/test_cgroupstelemetry.py

```diff
@@ -619,9 +620,8 @@ def test_extension_temetry_not_sent_for_empty_perf_metrics(self, *args):
 
     @skip_if_predicate_false(i_am_root, "This test will only run as root")
     @skip_if_predicate_false(are_cgroups_enabled, "Does not run when Cgroups are not enabled")
+    @skip_if_predicate_true(is_trusty_in_travis, "Does not run on Trusty in Travis")
```

Comment:
why this test does not run on Trusty in Travis?

Reply:
I added a comment on that function explaining why: https://github.com/Azure/WALinuxAgent/pull/1592/files/6b2db46027037dad5e2095aac8cc6a6716d19448?file-filters%5B%5D=.py#diff-ed85166e54c2abaa1e8a2fcc867ad749R144